brave/brave-ios
# Fix #7855: Add 'Private tabs' settings section

## Summary of Changes

This PR also organize and arrange the views and controller to desired folders for settings.


This pull request fixes #7855

## Submitter Checklist:


## Test Plan:
- Verify Private tabs under Tab section
- Verify Private tabs include Persistent Private Tabs toggle and Local Authentication Toggle
- Verify toggles are working as intended

QA: Search for `Private Tab Mode Local Authentication Cases` for additional information about cases

## Screenshots:


https://github.com/brave/brave-ios/assets/6643505/7dae955b-836b-43b2-9249-58bbf0943d44




![111](https://github.com/brave/brave-ios/assets/6643505/bb342854-77a2-4725-bc2c-02e8a9170ed1)


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
